templates/README.txt

```diff
@@ -1,6 +1,6 @@
 This folder is where you should place all your overriding template files. By
-default, the Bootstrap base theme provides all the necessary template files in
-various folders inside of sites/*/themes/bootstrap/theme. For example, the
+default, the Atomium base theme provides all the necessary template files in
+various folders inside of sites/*/themes/atomium/templates. For example, the
 page.tpl.php template file is located at
-sites/*/themes/bootstrap/theme/system/page.tpl.php. To override any of these
-files, copy them from the Bootstrap base theme and place them in here.
+sites/*/themes/atomium/templates/page/page.tpl.php. To override any of these
+files, copy them from the Atomium base theme and place them in here.
```

Comment:
I suggest to put a reference like "For more information, please read the readme file of theatomium theme"